### ISSUE TITLE
Positions don't seem to be saved for 'default' window

### ISSUE THREAD

**Issue by Sakata-MC** [Phrynohyas/eve-o-preview#141]
The default character-select preview renders at 0,0, and doesn’t matter if I save the window elsewhere either positive or negative value. 
What I’d like to happen is that is doesn’t cover up my main window while I’m logging in.

**Comment by epoch18** [comment]
Can confirm. I see the setting in the config file, but no matter what I set, the default (character selection) windows always appear in the same place.

**Comment by Sakata-MC** [comment]
#96 
Should we close this issue as it appears you may have this on your radar?

**Comment by Phrynohyas** [comment]
Nope. I'll close them both once this is implemented. For consistency reasons

**Comment by epoch18** [comment]
Hey man. Just checking in. Any progress on resolving this issue? 
I appreciate your work. Thanks.

**Comment by C-O-dE** [comment]
ignore the 175 link, that was the wrong screen
